<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleInvProjAnalysisModule</class>
 <widget class="QMainWindow" name="SingleInvProjAnalysisModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QGroupBox" name="theoreticalAnalysisGroupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="theoreticalAnalysisCheckBox">
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>158</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="ppCheckBox">
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>158</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="arrCheckBox">
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>158</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="restCostValueLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>139</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="3">
        <widget class="QDoubleSpinBox" name="restCostValueDoubleSpinBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QGroupBox" name="realAnalysisGroupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="discountRateLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>218</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QDoubleSpinBox" name="discountRateDoubleSpinBox"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="piCheckBox">
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>217</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="npvCheckBox">
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>217</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="irrCheckBox">
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2">
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>217</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QCheckBox" name="dppCheckBox">
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="2">
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>217</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QGroupBox" name="invRisksGroupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="invRisksCheckBox">
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="4">
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>545</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="sensetivityAnalysisCheckBox">
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="4">
        <spacer name="horizontalSpacer_11">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>545</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_12">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>179</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLabel" name="sensetivityAnalysisFactorsLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <spacer name="horizontalSpacer_13">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>257</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <spacer name="horizontalSpacer_14">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>179</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLabel" name="sensetivityIntervalLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3" colspan="2">
        <spacer name="horizontalSpacer_15">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>388</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="5">
        <widget class="QDoubleSpinBox" name="sensetivityIntervaDoubleSpinBox"/>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QCheckBox" name="bepCheckBox">
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="4">
        <spacer name="horizontalSpacer_16">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>545</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QPushButton" name="scenarioAnalysisPushButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="4">
        <spacer name="horizontalSpacer_17">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>545</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="monteKarloAnalysisPushButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2" colspan="4">
        <spacer name="horizontalSpacer_18">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>545</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="4" colspan="2">
        <widget class="QPlainTextEdit" name="sensetivityAnalysisFactorsPlainTextEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <spacer name="horizontalSpacer_19">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>619</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="singleInvProjAnalysePushButton">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
